plenum/server/node.py

```diff
@@ -3259,7 +3264,13 @@ def three_phase_key_for_txn_seq_no(self, ledger_id, seq_no):
 
         if ledger_id in self.txn_seq_range_to_3phase_key:
             # point query in interval tree
-            s = self.txn_seq_range_to_3phase_key[ledger_id][seq_no]
+            tuple = self.txn_seq_range_to_3phase_key[ledger_id]
```

Comment:
`tuple` is a type name, so it's better to avoid using it as a variable name.

Reply:
sure